packages/backend/src/graphql/resolvers/outgoing_payment.ts

```diff
@@ -248,6 +249,20 @@ export function paymentToGraphql(
     createdAt: new Date(+payment.createdAt).toISOString(),
     quote: quoteToGraphql(payment.quote),
     grantId: payment.grantId,
-    tenantId: payment.tenantId
+    tenantId: payment.tenantId,
+    cardDetails: cardToGraphql(payment)
+  }
+}
+function cardToGraphql(
+  payment: OutgoingPayment
```

Comment:
nitpick: If this function is called `cardToGraphql`, perhaps the input should be `CardDetails` instead of `OutgoingPayment`